react-toggle-switch-component/dist/styles.css

```diff
@@ -0,0 +1,61 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.background {
+  height: 100vh;
+  transition: 0.5s;
+}
+
+.background-dark {
+  background: #2f2f2f;
+}
+
+.toggle-switch {
+  display: flex;
+  align-items: center;
+  padding: 1rem;
+}
+
+input[type="checkbox"] {
+  position: relative;
+  width: 80px;
+  height: 40px;
+  -webkit-appearance: none;
+  background: rgb(221, 221, 221);
+  outline: none;
+  border-radius: 20px;
+  transition: 0.5s;
+}
+
+input:checked[type="checkbox"] {
+  background: rgb(19, 101, 209);
+}
+
+input[type="checkbox"]::before {
+  content: '';
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  border-radius: 20px;
+  top: 0;
+  left: 0;
+  background: #fff;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+  transition: 0.5s;
+}
+
+input:checked[type="checkbox"]::before {
+  left: 40px;
+}
```

Comment:
If you can explain this CSS to me, I will approve to merge! You can send me your explanation via Slack.